### DISCUSSION TITLE
Clustering using rnmapbox/maps

### DISCUSSION THREAD

**Discussion by andreipatza** [rnmapbox/maps#2526] [General]
Hello guys, 

I'm trying to do point clustering using the new version of rnmapbox/maps in react-native. 

So far, I can not achieve the goal because I'm kind of stuck between documentation and examples found on the internet. 

Can one of you help me a little with this? 

Thank you in advice.

**Comment by mukuljeswani** [comment]
Hi andreipatza

Did you find documentation for clustering on v10 ?

**Comment by two3d** [comment]
You can find a doc here: https://rnmapbox.github.io/docs/components/ShapeSource
And here: https://github.com/rnmapbox/maps/blob/main/docs/ShapeSource.md
